app/models/manageiq/providers/kubevirt/infra_manager.rb

```diff
@@ -53,6 +53,41 @@ def self.description
     @description ||= ManageIQ::Providers::Kubevirt::Constants::PRODUCT
   end
 
+  def self.params_for_create
+    @params_for_create ||= {
+      :title  => "Configure Kubevirt",
+      :fields => [
+        {
+          :component  => "text-field",
+          :name       => "endpoints.default.server",
+          :label      => "Hostname",
+          :isRequired => true,
+          :validate   => [{:type => "required-validator"}]
+        },
+        {
+          :component  => "text-field",
+          :name       => "endpoints.default.port",
+          :type       => "number",
+          :isRequired => true,
+          :validate   => [{:type => "required-validator"}]
```

Comment:
I think there's a validator for numbers.

Reply:
You mean for a port range like `[1, 65535]`?  Yeah can add that